lib/dfe_sign_in_api.rb

```diff
@@ -0,0 +1,35 @@
+module DFESignIn
+  class API
+    BASE_URL = DFE_SIGN_IN_URL
+
+    attr_accessor :dfe_sign_in_password
+
+    def initialize
+      self.dfe_sign_in_password = DFE_SIGN_IN_PASSWORD
+    end
+
+    def users(page: 1)
+      token = generate_jwt_token
+      response = HTTParty.get(
```

Comment:
How does this gateway behave if there is a 500 error from the remote service?

Can this API return any other response code?

Reply:
Updated in 51186ec to tolerate these errors